packages/react/src/callout/docs/overview.mdx

```diff
@@ -73,7 +73,7 @@ Use an `info` tone to highlight information which is important for users.
 
 ## Compact
 
-Compact version of the callout can be used when there is limited space, such as inside of a Control group to let the user know that, based on their response, they may need to provide additional information.
+Compact version of the callout can be used when there is limited space, such as inside of an Accordion to let the user know that, based on their response, they may need to provide additional information.
```

Comment:
I would remove everything passed "Accordion" because it mentions stuff about "response"